### PR TITLE
Address issues are fixed in order confirmation email.

### PR DESCRIPTION
**Pull request by anchal-a**
Confirmation mail address issues are fixed. Kindly review and merge.

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/etc/notification/email/hclecomm/order-confirmation-email-template.html [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
+Total Amount: $ ${grandTotal}
 
 Shipping Address: ${address}
 
 
 Manage Your Orders At: http://localhost:4502/content/hclecomm/us/en/orders.html
 
-Click on this link to continue shopping with us!! 
+Click on this link to continue shopping with us!!
 http://localhost:4502/content/hclecomm/us/en/home.html
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/etc/notification/email/hclecomm/order-confirmation-email-template.html:18]: @anchal-a : can we remove the absolute URL path & make it relative.

File: ui.apps/src/main/content/jcr_root/etc/notification/email/hclecomm/order-confirmation-email-template.html [hunk 2]
```diff
@@ -7,14 +7,14 @@
 
 Your Order ID is ${orderId}.
 
-Total Amount:${grandTotal}
+Total Amount: $ ${grandTotal}
 
 Shipping Address: ${address}
 
 
 Manage Your Orders At: http://localhost:4502/content/hclecomm/us/en/orders.html
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/etc/notification/email/hclecomm/order-confirmation-email-template.html:15]: @anchal-a : can we remove the absolute URL path & make it relative.

**Review by rishabh-behl — COMMENTED** [review]
@anchal-a : pl incorporate changes

File: ui.apps/src/main/content/jcr_root/etc/notification/email/hclecomm/order-confirmation-email-template.html (hunk 1 above)
Comment (anchal-a) [reply ui.apps/src/main/content/jcr_root/etc/notification/email/hclecomm/order-confirmation-email-template.html:18]: Yes Rishabh, This bug is under process. It needs to be done in all the templates. So will fix it later in all the templates.

**Review by rishabh-behl — APPROVED** [review]
ok

**Pull request merged**